packages/flutter_template_images/LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright 2020 The Flutter Authors. All rights reserved.
```

Comment:
please use the  same year as the flutter/flutter license so that it dedupes (or can dedupe... i don't know if we actually implement that logic for packages yet)

Reply:
Will do. There wasn't any consistency to the rest of the packages here so I wasn't sure if we were doing that.

Reply:
yeah this repo is a bit of a neglected child, we haven't done much to enforce the same kinds of things as in other repos. we should though.